ush/config_defaults.sh

```diff
@@ -1569,6 +1569,7 @@ WTIME_GET_EXTRN_LBCS="00:45:00"
 WTIME_MAKE_ICS="00:30:00"
 WTIME_MAKE_LBCS="01:30:00"
 WTIME_RUN_PREPSTART="00:10:00"
+WTIME_RUN_PREPSTART_ENSMEAN="00:30:00"
```

Comment:
Just a thought, can we just increase WTIME_RUN_PREPSTART to 30 minutes?

Reply:
Would it impact the real-time run? If not, we can have at least one less variable here.

Reply:
It will not affect the real-time runs as they uses reservation nodes.  Also I think 30minutes vs 10 minutes wall time requests may not make big impact in the slurm scheduling process.